app/js/lib/foxbox/settings.js

```diff
@@ -121,13 +111,19 @@ export default class Settings extends Model {
   }
 
   set session(session) {
-    if (session === undefined) {
-      this._session = undefined;
-      storage.removeItem(`${PREFIX}session`);
-    } else {
-      this._session = session;
+    if (this._session === session) {
+      return;
+    }
+
+    this._session = session;
+
+    if (this._session) {
       storage.setItem(`${PREFIX}session`, this._session);
+    } else {
+      storage.removeItem(`${PREFIX}session`);
```

Comment:
as far as I can tell, we should always remove session from localstorage if ```session``` is not non-empty string, that's why I'm changing this. Am I missing something?